exams/signals_test.py

```diff
@@ -38,7 +38,11 @@ def setUpTestData(cls):
         with mute_signals(post_save):
             cls.profile = profile = ProfileFactory.create()
 
-        program, _ = create_program(past=True)
+        program, _ = create_program(
+            past=True,
+            exam_series_code="TestExam",
+            exam_module="SampleCourse"
+        )
```

Comment:
I understand, but this file is `signal_test` and you have not modified the file `signal`, so why there is a change here?